<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupcompanywindow</class>
 <widget class="QDialog" name="setupcompanywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setup Company</string>
  </property>
  <widget class="QLabel" name="companyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Setup Your Company:</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyLabelEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="inChargeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Who is in charge and how many people report to him/her?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setLeaderButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans MT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLabel" name="addEmployeesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Employees Reporting To:</string>
   </property>
  </widget>
  <widget class="QLabel" name="leaderNameLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>225</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LeaderName</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="employeeNameLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="employeePhoneLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="leaderNameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leaderPhoneLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leaderNCLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="inChargeNameLabel">
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Leader First Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="inChargePhoneLabel">
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Leader Phone Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="leaderNCLabel">
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of employees managed: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="employeeNameLabel">
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Names: (comma separated: a,b,c..)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="employeePhoneLabel">
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone Numbers: (comma separated)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>293</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans MT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Employees</string>
   </property>
  </widget>
  <widget class="QLabel" name="promptLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>prompt!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans MT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
